packages/admin/cms-admin/src/pages/pagesPage/PagesPage.tsx

```diff
@@ -220,11 +221,35 @@ export function PagesPage({
                             const page = data?.pages.find((page) => page.id == selectedId);
 
                             if (!page) {
-                                return null;
+                                return (
+                                    <Box px={4} py={4}>
+                                        <Alert
+                                            title={<FormattedMessage id="comet.pages.pages.notFound" defaultMessage="Not found" />}
+                                            severity="error"
+                                        >
+                                            <FormattedMessage
+                                                id="comet.pages.pages.documentDoesntExist"
+                                                defaultMessage="This document doesn't exist"
+                                            />
+                                        </Alert>
+                                    </Box>
+                                );
                             }
 
                             if (page.visibility === "Archived") {
-                                return <>403, not allowed</>;
+                                return (
+                                    <Box px={4} py={4}>
+                                        <Alert
+                                            title={<FormattedMessage id="comet.pages.pages.archived" defaultMessage="Archived" />}
+                                            severity="warning"
```

Comment:
IMO this is an error as well

Reply:
I made it a warning because the user can still do something about it.

But maybe you are right. A warning usually means you can proceed but it's not recommended. Here you can't do that